packages/plugins/Traderxyz/src/SNSAdaptor/components/TreeParentContent.tsx

```diff
@@ -0,0 +1,57 @@
+import { TreeItemContentProps, useTreeItem } from '@mui/lab/TreeItem'
+import { Avatar, Fade, Typography, AvatarGroup } from '@mui/material'
+import * as React from 'react'
+import clsx from 'clsx'
+
+const TreeParentContent = React.forwardRef(function CustomContent(
```

Comment:
Why do you need forwardRef? Looks like you're not using it.

Reply:
the ref is forwarded to the component to handle the expansion of item as mentioned by mui as seen below.
<img width="473" alt="image" src="https://user-images.githubusercontent.com/3579069/165043796-dd5488f4-224e-4803-b4c4-d041ee3f789c.png">

Reply:
Fill the generics of forwardRef.